getting_started/dev-environment/macOS.md

```diff
@@ -0,0 +1,73 @@
+# Instructions for macOS
+
+Install [Homebrew](http://brew.sh/) to give your mac a package manager. Open up your terminal emulator of choice and run:
+
+```shell
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+Install some basic command line tools by running:
+
+```shell
+$ brew install coreutils
+```
+
+Install git:
+
+```shell
+$ brew install git
+```
+
+Install python and its package manager, pip:
+
+```shell
+$ brew install python
+$ curl -O http://python-distribute.org/distribute_setup.py
+$ python distribute_setup.py
+$ curl -O https://raw.github.com/pypa/pip/master/contrib/get-pip.py
+$ python get-pip.py
+$ rm distribute_setup.py get-pip.py
+```
+
+Install virtualenv and virtualenvwrapper:
+
+```shell
+$ pip install virtualenv virtualenvwrapper
+```
+
+Add the following to your `~/.bash_profile`:
+
+```shell
+# Create a backup of your .bash_profile
+cp ~/.bash_profile ~/.bash_profile-org
+
+# Be careful with this command
+printf '\n%s\n%s\n%s' '# virtualenv' 'export WORKON_HOME=~/virtualenvs' \
+'source /usr/local/bin/virtualenvwrapper.sh' >> ~/.bash_profile
```

Comment:
This should be the same as the docs for ubuntu:
```markdown
    ```shell
    export WORKON_HOME=~/.virtualenvs
    . /usr/local/bin/virtualenvwrapper.sh
    ```
```